search.js

```diff
@@ -1,56 +1,220 @@
-  { name: "Cafe Rio", address: "253 17th St, Costa Mesa, CA 92627", price: "$", distance: "0.7 mi", phone: "(949) 335-6800", rating: "rating: 3", review: "Is there even an interview here?"},
-  { name: "Pitfire Artisan Pizza", address: "4353 E 17th St, Costa Mesa, CA 92627", price: "$$", distance: "1.2 mi", phone: "(949) 313-6333", rating: "rating: 4", review: "Yum. Yum. Yum. Yum"},
-  { name: "Avila's El Ranchito", address: "2101 Placentia Ave, Costa Mesa, CA 92627", price: "$$", distance: "0.4 mi", phone: "(949) 642-1142", rating: "rating: 4", review: "Bomb old school mexican. I think the whole family works there." },
-  { name: "McDonald's", address: "2300 Harbor Blvd, Costa Mesa, CA 92626", price: "$", distance: "2.3 mi", phone: "(949) 631-1943", rating: "rating: 2", review: "There's homeless people outside, stick to the drive-thru."}
+  { name: "Ruth Chris",
+    address: "2961 Michelson Dr #A, Irvine, CA 92612",
+    price: "$$$$",
+    distance: "6.9 mi",
+    phone: "(949)252-8848",
+    rating: "rating: 4",
+    review: "Butter on the steak!"},
+
+  { name: "Mastro's Steakhouse",
+    address: "633 Anton Blvd, Costa Mesa, CA 92626",
+    price: "$$$$",
+    distance: "5.2 mi",
+    phone: "(714) 546-7405",
+    rating: "rating: 4",
+    review: "Great food, poor service."},
+
+  { name: "Roy's Hawaiian Fusion",
+    address: "453 Newport Center Dr, Newport Beach, CA 92660",
+    price: "$$$$",
+    distance: "5.7mi",
+    phone: "(949) 640-7697",
+    rating: "rating: 4",
+    review: "My favorite place! Lobster potstickers...hello heaven!"},
+
+  { name: "Cafe Rio",
+    address: "253 17th St, Costa Mesa, CA 92627",
+    price: "$",
+    distance: "0.7 mi",
+    phone: "(949) 335-6800",
+    rating: "rating: 3",
+    review: "Is there even an interview here?"},
+
+  { name: "Pitfire Artisan Pizza",
+    address: "4353 E 17th St, Costa Mesa, CA 92627",
+    price: "$$",
+    distance: "1.2 mi",
+    phone: "(949) 313-6333",
+    rating: "rating: 4",
+    review: "Yum. Yum. Yum. Yum"},
+
+  { name: "Avila's El Ranchito",
+    address: "2101 Placentia Ave, Costa Mesa, CA 92627",
+    price: "$$",
+    distance: "0.4 mi",
+    phone: "(949) 642-1142",
+    rating: "rating: 4",
+    review: "Bomb old school mexican. I think the whole family works there." },
+
+  { name: "McDonald's",
+    address: "2300 Harbor Blvd, Costa Mesa, CA 92626",
+    price: "$",
+    distance: "2.3 mi",
+    phone: "(949) 631-1943",
+    rating: "rating: 2",
+    review: "There's homeless people outside, stick to the drive-thru."}
 ]
 
+//disabling searching by zipcode
 var zipSearch = document.getElementById("location");
 zipSearch.setAttribute("disabled", true);
 
+//loading each object in the businesses array on the search page
+//puting it inside a new div called business
 for (var i = 0; i < businesses.length; i++) {
-  var business = document.createElement("div")
+  var business = document.createElement("div");
+  business.setAttribute("id", "business");
```

Comment:
HTML `id` attributes should be unique page-wide. For styling, use a CSS class instead. For identification, use a unique value from your data model.
<img width="444" alt="screenshot 2016-11-24 13 35 03" src="https://cloud.githubusercontent.com/assets/7432943/20610186/f8dcded4-b24a-11e6-9475-e84910ead8a9.png">